unordered_associative_task_1/student/include/impl.h

```diff
@@ -32,4 +32,24 @@ using playlist = std::vector<Song>;
  *
  * @return true if first playlist is anagram of second
  */
-bool is_same_content(const playlist& first_playlist, const playlist& second_playlist);
+bool is_same_content(const playlist& first_playlist, const playlist& second_playlist){
+    std::unordered_map<std::string, int> songs;
+    
+    //POPULATING MAP
+    for(auto& s : first_playlist){
+        if(songs.find(s.name()) == songs.end()) songs.insert({s.name(), 1});
```

Comment:
please make your statements visible, separate it by multiple lines and use {} after your statement